bin/fenix-qr-codes.js

```diff
@@ -20,34 +22,36 @@ if (!fs.existsSync(destDir)) {
 
 (async () => {
   const res = await fetch(
-    `https://addons.mozilla.org/api/v5/addons/search/?app=android&promoted=recommended&type=extension&lang=${lang}`
+    `https://addons.mozilla.org/api/v5/addons/search/?app=android&promoted=recommended&type=extension&lang=${lang}`,
   );
   const { results: addons } = await res.json();
 
   await Promise.all(
     addons.map(({ id, url }) => {
-      const filePath =   path.join(destDir, `${id}.png`);
+      const filePath = path.join(destDir, `${id}.png`);
       let content = url.replace(`/${lang}`, '');
       content = [
         `${content}?utm_campaign=amo-fenix-qr-code`,
         `utm_content=${id}`,
         `utm_medium=referral&utm_source=addons.mozilla.org`,
-      ].join('&')
-      return QRCode.toFile(filePath, content, {margin: 1});
-    })
+      ].join('&');
+      return QRCode.toFile(filePath, content, { margin: 1 });
+    }),
   );
 
-  const knownIds = addonIdsWithQRCodes.addonIds;
-  const addonIdsFromAPI = addons.map(({ id }) => id);
+  console.log(addons.length, ' QR codes created');
 
-  for (const addonId of addonIdsFromAPI) {
-    if (!knownIds.includes(addonId)) {
-      console.log(`'addonIdsWithQRCodes' might be outdated:`);
-      console.log(`  addonIdsWithQRCodes=${knownIds.sort()}`);
-      console.log(`  IDs from API=${addonIdsFromAPI.sort()}`);
-      process.exit(1);
-    }
+  const knownIds = config.get('addonIdsWithQRCodes');
+  const addonIdsFromAPI = [...new Set(addons.map(({ id }) => id))];
```

Comment:
Do we really need all this `Set` stuff? Can we not just use `deep-eql`?

Reply:
The `Set` combined with the array spread is to end up with an array without duplicates. That's needed, whether we use `deep-eql` or not. We could use `deep-eql` a few lines below, though.